sys-libs/liburing/liburing-2.3-r2.ebuild

```diff
@@ -39,6 +39,13 @@ src_prepare() {
 		sed -i "/^Version:/s@[[:digit:]\.]\+@${PV}@" ${PN}.spec || die
 	fi
 
+	if ! use examples; then
```

Comment:
Should we force this by adding something like ```if use elibc_musl || ! use examples.; then```?

Reply:
It's a good question, but it's something we should do in `profiles/` instead (in `profiles/features/musl/package.use.mask`).